### DISCUSSION TITLE
Shaders file to be uploaded with source file

### DISCUSSION THREAD

**Discussion by pym09** [KhronosGroup/Vulkan-Samples#554] [General]
Hi,

Can the shader files be added along with the source cpp files. It will help most of the beginners who are looking for samples

**Comment by SaschaWillems** [comment]
Can you elaborate? All GLSL shaders are part of the repository.

**Reply by pym09** [reply]
Sorry my bad. I was looking for the shader files within the source file folders. I actually found them in another separate folder "Shaders". Thanks for the response.
